pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -3257,7 +3375,7 @@ class MachineLearningPatch(Patch, object):
     frequency : float, None
         The patch frequency, it is used in prediction formulas. If it is None, the patch frequency will be that of the
         layer or of the stackup. From 0.1 to 10 GHz.
-    patch_width : float
+    dx : float
```

Comment:
None of these descriptions follow our guidelines either. Someday I hope to find some time to do an overall edit of the PyAEDT doc, section by section.

Reply:
Good to know. I was using existing comments as a reference.